### PR TITLE
Convert FD number type to SQL numeric type

### PR DESCRIPTION
**Pull request by roll**

### CODE REVIEW

File: tableschema_sql/mapper.py [hunk 1]
```diff
@@ -127,14 +127,20 @@ def convert_type(self, type):
...
+            'number': sa.Numeric,
             'object': None,
             'string': sa.Text,
             'time': sa.Time,
             'year': sa.Integer,
             'yearmonth': None,
         }
 
+        # Mysql dialect
+        if self.__dialect == 'mysql':
```
Comment (roll) [inline tableschema_sql/mapper.py:139]: @akariv 
I have no idea why but using `sa.Numeric` totally screw MySQL in tests. Like rounding numbers and switching `0/1` to `False/True`

Comment (akariv) [reply tableschema_sql/mapper.py:139]: So maybe just for psql?

Comment (roll) [reply tableschema_sql/mapper.py:139]: It works with `sa.Numeric` for Postgres/SQLite and here I use fallback for MySQL

**Review by akariv — APPROVED** [review]
+1

**Pull request merged**